script/validate_builds.py

```diff
@@ -81,7 +81,7 @@ def trigger_script_validation_checks(file_name, image_name = "registry.access.re
         stderr = True, # Return logs from STDERR
     )
     result = container.wait()
-    print(container.logs())
+    print(container.logs().decode("utf-8"))
```

Comment:
@kandarpamalipeddi can you point to a sample travis build? Which shows the readable format of logs, compared to current log format.

Reply:
I checked the changes on local python script with container execution.

Below is the python script that used before to print the contents:

```
import docker

image_name = "registry.access.redhat.com/ubi8/ubi:8.5"
client = docker.DockerClient(base_url='unix://var/run/docker.sock')
container = client.containers.run(
        image_name,
        "cat /etc/os-release",
        network = 'host',
        detach = False,
        stdout = True,
        stderr = True, # Return logs from STDERR
        auto_remove = True
    )
print(container)
```

This gave me below output:
```
#  python travis.py
b'NAME="Red Hat Enterprise Linux"\nVERSION="8.5 (Ootpa)"\nID="rhel"\nID_LIKE="fedora"\nVERSION_ID="8.5"\nPLATFORM_ID="platform:el8"\nPRETTY_NAME="Red Hat Enterprise Linux 8.5 (Ootpa)"\nANSI_COLOR="0;31"\nCPE_NAME="cpe:/o:redhat:enterprise_linux:8::baseos"\nHOME_URL="https://www.redhat.com/"\nDOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"\nBUG_REPORT_URL="https://bugzilla.redhat.com/"\n\nREDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"\nREDHAT_BUGZILLA_PRODUCT_VERSION=8.5\nREDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"\nREDHAT_SUPPORT_PRODUCT_VERSION="8.5"\n'
```

Updated the script as below :
```
import docker

image_name = "registry.access.redhat.com/ubi8/ubi:8.5"
client = docker.DockerClient(base_url='unix://var/run/docker.sock')
container = client.containers.run(
        image_name,
        "cat /etc/os-release",
        network = 'host',
        detach = False,
        stdout = True,
        stderr = True, # Return logs from STDERR
        auto_remove = True
    )
print(container.decode("utf-8"))
```
Below is the output:
```
# python travis.py
NAME="Red Hat Enterprise Linux"
VERSION="8.5 (Ootpa)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="8.5"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Red Hat Enterprise Linux 8.5 (Ootpa)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:redhat:enterprise_linux:8::baseos"
HOME_URL="https://www.redhat.com/"
DOCUMENTATION_URL="https://access.redhat.com/documentation/red_hat_enterprise_linux/8/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"

REDHAT_BUGZILLA_PRODUCT="Red Hat Enterprise Linux 8"
REDHAT_BUGZILLA_PRODUCT_VERSION=8.5
REDHAT_SUPPORT_PRODUCT="Red Hat Enterprise Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.5"

```

Added build-script created for `cli-table3` #1620, and I saw proper output with python script with decode("utf-8").